guides/common/modules/ref_global-parameters-for-registration.adoc

```diff
@@ -20,4 +20,4 @@ The default value is `true`.
 * The `remote_execution_ssh_keys`, `remote_execution_ssh_user`, `remote_execution_create_user`, and `remote_execution_effective_user_method` parameters are used in the `remote_execution_ssh_keys` snippet.
 For more details, see the snippet.
 
-You can navigate to snippets in the {ProjectWebUI} through *Hosts* > *Templates* > *Provisioning Templates*.
\ No newline at end of file
+You can navigate to snippets in the {ProjectWebUI} through *Hosts* > *Templates* > *Provisioning Templates*.
```

Comment:
just FYI; this is what I meant @mjivraja 

my local git diff:

```` diff
-You can navigate to snippets in the {ProjectWebUI} through *Hosts* > *Templates* > *Provisioning Templates*.
\ No newline at end of file
+You can navigate to snippets in the {ProjectWebUI} through *Hosts* > *Templates* > *Provisioning Templates*.
````

Reply:
Oww yes. Thank you for sharing @maximiliankolb :star: